nncf/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -264,6 +264,10 @@ def get_weight_nodes(nncf_graph: NNCFGraph) -> List[NNCFNode]:
             if isinstance(node.layer_attributes, OVLayerAttributes) and node.metatype in OPERATIONS_WITH_WEIGHTS
         ]
 
+    @staticmethod
+    def is_constant_matmul(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        return node.layer_attributes is not None
```

Comment:
The same here. Refer to [nncf/openvino/graph/nncf_graph_builder.py](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/openvino/graph/nncf_graph_builder.py) find other metatypes. The name of the method is incorrect.